### ISSUE TITLE
when change to 1.2.0，got full noised image output

### ISSUE THREAD

**Issue by taotaow** [DarthAffe/StableDiffusion.NET#8]
when changed to StableDiffusion.NET.Backend.Cpu 1.2.0
got full noised image output
stabledifffusion.net.backend.cpu.1.1.0 version is good

**Comment by DarthAffe** [comment]
i can't reproduce that.
are you using any loras? in that case this might be related:
https://github.com/leejet/stable-diffusion.cpp/issues/242

if not, does the same issue occur when you run the tool provided by stable-diffusion.cpp (sd.exe) with the exact same parameters?

**Comment by taotaow** [comment]
yes,I used lcm lora.I check the log,and see large amout skip lora errors like below:
model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_proj_in.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_proj_out.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_k.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_out_0.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_q.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_v.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn2_to_k.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn2_to_out_0.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn2_to_q.alpha' with n_dims 0

model.cpp:893  - skip tensor 'lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn2_to_v.alpha' with n_dims 0

**Comment by taotaow** [comment]
should apply this commit to fix this problem:
https://github.com/leejet/stable-diffusion.cpp/commit/760cfaa61896a02dab8b1ac84a59b17ddf951090

**Comment by DarthAffe** [comment]
this commit is included in the 1.2 backends (they are based on the currently latest release [master-ce1bcc7](https://github.com/leejet/stable-diffusion.cpp/releases/tag/master-ce1bcc7))
but it seems like this fix somehow caused the issue, but i don't know any more about this than what's written in the issue over there,

**Comment by taotaow** [comment]
master-6d16f68 is ok
master-760cfaa commit https://github.com/leejet/stable-diffusion.cpp/commit/760cfaa61896a02dab8b1ac84a59b17ddf951090 cause the bug
master-ce1bcc7 keep that bug and add skip tensor log.

**Comment by DarthAffe** [comment]
there are new backend-packages available (1.2.1) that contain the [latest release](https://github.com/leejet/stable-diffusion.cpp/releases/tag/master-1d2af5c) of stable-diffusion.cpp which should fix the issue.

**Comment by taotaow** [comment]
yes,1.2.1fix the issue
thank you
